google/cloud/spanner/client_test.cc

```diff
@@ -981,6 +981,27 @@ TEST(ClientTest, CommitStats) {
   EXPECT_EQ(42, result->commit_stats->mutation_count);
 }
 
+TEST(ClientTest, MaxCommitDelay) {
+  auto timestamp =
+      spanner_internal::TimestampFromRFC3339("2020-10-20T02:20:09.123Z");
+  ASSERT_STATUS_OK(timestamp);
+
+  auto conn = std::make_shared<MockConnection>();
+  EXPECT_CALL(*conn, Commit)
+      .WillOnce([&timestamp](Connection::CommitParams const& cp) {
+        EXPECT_EQ(cp.options.max_commit_delay(),
+                  std::chrono::milliseconds(100));
+	return CommitResult{*timestamp, absl::nullopt};
```

Comment:
It looks like you have a TAB there instead of spaces.

Reply:
Done.